includes/Database_Upgrader.php

```diff
@@ -274,6 +275,59 @@ protected function rewrite_flush() {
 		}
 	}
 
+	/**
+	 * Re-index stories in Yoast SEO if permalinks are outdated.
+	 *
+	 * @return void
+	 */
+	protected function yoast_reindex_stories() {
+		if (
+			! function_exists( 'YoastSEO' ) ||
+			! class_exists( '\Yoast\WP\SEO\Repositories\Indexable_Repository' ) ||
+			! class_exists( '\Yoast\WP\SEO\Builders\Indexable_Builder' ) ||
+			! is_a( YoastSEO(), '\Yoast\WP\SEO\Main' )
+		) {
+			return;
+		}
+
+		/**
+		 * Indexable Repository.
+		 *
+		 * @var \Yoast\WP\SEO\Repositories\Indexable_Repository $repository Indexable Repository.
+		 */
+		$repository = YoastSEO()->classes->get( 'Yoast\WP\SEO\Repositories\Indexable_Repository' );
+
+		/**
+		 * Indexable Builder.
+		 *
+		 * @var \Yoast\WP\SEO\Builders\Indexable_Builder $builder Indexable Builder.
+		 */
+		$builder = YoastSEO()->classes->get( 'Yoast\WP\SEO\Builders\Indexable_Builder' );
+
+		$indexable_before = $repository->find_for_post_type_archive( Story_Post_Type::POST_TYPE_SLUG, false );
+
+		if ( ! $indexable_before || false === strpos( $indexable_before->permalink, '/web-stories//' ) ) {
```

Comment:
Why does this line have two slashes in it?

Reply:
Typo